database.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE IF NOT EXISTS `Database shizzle`;
+
+CREATE TABLE `table enzo `(
```

Comment:
you can also type CREATE TABLE IF NOT EXISTS like you did above with create database

Reply:
Ok cool thank you for the tip i will try it out

Reply:
As Filip said now, the if not exists is not useful in this case. But also syntax says to not have spaces like in your table name. Maybe better to just name it enzo.